### ISSUE TITLE
If images are added to an album after the album was viewed in Slide, they won't appear

### ISSUE THREAD

**Issue by touzainanboku** [ccrama/Slide#2256]
Slide version: 5.5
Android version: 6.0.1

If you open an album in Slide, then the creator of the album adds new images to it, you won't see those new images. 

To reproduce it:
- Create an album on Imgur (for example 2 images)
- Open that album in Slide (for example by PMing the link to yourself) 
- Add new images to the album (for example 1 image)
- Open the album again in Slide: you'll only see the original 2 images

If you clear Slide's data then it'll show you 3 images. Clearing cache didn't help. 

If (for whatever reason) it's not feasible to check the number of images in an album every time, is it possible to add a button or something that forces a refresh of some sorts?

**Comment by ccrama** [comment]
That is because Slide caches albums to access them quicker/not use more of the Imgur API. Don't know if it's worth the trade off to change that

**Comment by touzainanboku** [comment]
Yeah, I agree with that. So what's your thought on adding a button in the action overflow button to force a refresh if you ever need it? Although I totally understand if you wouldn't wanna do this since you're paying for Imgur's API.

**Comment by ccrama** [comment]
That might be worth it, I'll look into that. Don't know how the user would know new images were added, though

**Comment by touzainanboku** [comment]
Well usually they wouldn't know unless they opened the album with a different app and checked, but occasionally I've seen people comment that they've updated the album.

**Comment by piyush-rvc** [comment]
@ccrama facing the same issue. Also, doesn't slide get open source exception for free unlimited API use?
